editor/src/clj/util/digestable.clj

```diff
@@ -218,6 +219,10 @@
   (digest! [value writer opts]
     (digest-tagged! 'LayoutResult (str value) writer opts))
 
+  Graphics$ShaderDesc
+  (digest! [value writer opts]
+    (digest-tagged! 'ShaderDesc (str value) writer opts))
+
```

Comment:
There are probably more efficient ways to do this, but I think it's fine for now.